git-branchless/src/revset/eval.rs

```diff
@@ -204,14 +207,35 @@ pub(super) fn eval_name(ctx: &mut Context, name: &str) -> EvalResult {
 }
 
 pub(super) fn eval_fn(ctx: &mut Context, name: &str, args: &[Expr]) -> EvalResult {
-    let function = FUNCTIONS
-        .get(name)
-        .ok_or_else(|| EvalError::UnboundFunction {
-            name: name.to_owned(),
-            available_names: FUNCTIONS.keys().sorted().copied().collect(),
-        })?;
-
-    function(ctx, name, args)
+    if let Some(function) = FUNCTIONS.get(name) {
+        return function(ctx, name, args);
+    }
+
+    let alias_key = format!("branchless.revsets.alias.{}", name);
+    let alias_template = ctx
+        .repo
+        .get_readonly_config()
+        .map_err(EvalError::OtherError)?
+        .get::<String, String>(alias_key)
+        .map_err(EvalError::OtherError)?;
```

Comment:
```suggestion
    let alias_template: String = ctx
        .repo
        .get_readonly_config()
        .map_err(EvalError::OtherError)?
        .get(alias_key)
        .map_err(EvalError::OtherError)?;
```

If you add a type annotation, you should be able to skip annotating the generic arguments manually.